packages/editor-ui/src/components/Modal.vue

```diff
@@ -1,63 +1,77 @@
 <template>
-	<div>
-		<el-drawer
-			v-if="drawer"
-			:direction="drawerDirection"
-			:visible="visible && visibleDrawer"
-			:size="drawerWidth"
-			:before-close="closeDrawer"
-			>
-			<template v-slot:title>
-				<slot name="header" />
-			</template>
-			<template>
-				<slot name="content"/>
-			</template>
-		</el-drawer>
-		<el-dialog
-			v-else
-			:visible="dialogVisible"
-			:before-close="closeDialog"
-			:title="title"
-			:class="{ 'dialog-wrapper': true, [size]: true }"
-			:width="width"
-			:show-close="showClose"
-			:custom-class="getCustomClass()"
-			append-to-body
-		>
-			<template v-slot:title>
-				<slot name="header" v-if="!loading" />
-			</template>
-			<div class="modal-content" @keydown.stop @keydown.enter="handleEnter" @keydown.esc="closeDialog">
-				<slot v-if="!loading"  name="content"/>
-				<div class="loader" v-else>
-					<n8n-spinner />
-				</div>
+	<el-dialog
+		:visible="visible"
+		:before-close="closeDialog"
+		:title="title"
+		:class="{'dialog-wrapper': true, 'center': center}"
+		:width="width"
+		:show-close="showClose"
+		:custom-class="getCustomClass()"
+		:style="styles"
+		append-to-body
+	>
+		<template v-slot:title>
+			<slot name="header" v-if="!loading" />
+		</template>
+		<div class="modal-content" @keydown.stop @keydown.enter="handleEnter" @keydown.esc="closeDialog">
+			<slot v-if="!loading"  name="content"/>
+			<div class="loader" v-else>
+				<n8n-spinner />
 			</div>
-			<el-row  v-if="!loading" class="modal-footer">
-				<slot name="footer" :close="closeDialog" />
-			</el-row>
-		</el-dialog>
-	</div>
+		</div>
+		<el-row  v-if="!loading" class="modal-footer">
+			<slot name="footer" :close="closeDialog" />
+		</el-row>
+	</el-dialog>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
 
-const sizeMap: {[size: string]: string} = {
-	xl: '80%',
-	lg: '70%',
-	m: '50%',
-	default: '50%',
-};
-
```

Comment:
I have given up on trying to standardize these sizes.. much easier to support custom configuration per modal...